### ISSUE TITLE
H tag size for contact molecule in sidefoot

### ISSUE THREAD

**Issue by sarahsimpson09** [cfpb/cfgov-refresh#1363]
H tag for the contact name is a different size than other sidefoot content, like related resources.

**Comment by KimberlyMunoz** [comment]
Can you include a screenshot?

**Comment by sarahsimpson09** [comment]
![enforcement sidefoot](https://cloud.githubusercontent.com/assets/9706023/12586505/93167c6a-c41e-11e5-8600-adf0b493bb7a.png)

**Comment by duelj** [comment]
The "payments to harmed consumers" seems like the more appropriate heading size for the side-foot (I think this is an H3?). We are ok with other heading sizes being used but we do want them to be consistent across multiple elements.

**Comment by KimberlyMunoz** [comment]
Found the issue!

We created two organisms for the contact-info organism: `main-contact-info.html` and `sidebar-contact-info.html`. 

The `sidebar-contact-info.html` include should have been included here in the sidebar, but instead this the markup for `main-contact-info.html` which should only be included in the pre-footer. I realize this might be tricky because both of these are included in the "sidefoot", but that's the issue here.
